lib/puppet_litmus.rb

```diff
@@ -8,11 +8,32 @@ module PuppetLitmus; end
 require 'puppet_litmus/puppet_helpers'
 require 'puppet_litmus/rake_helper'
 require 'puppet_litmus/spec_helper_acceptance'
+require 'honeycomb-beeline'
 
 # Helper methods for testing puppet content
 module PuppetLitmus
   include BoltSpec::Run
   include PuppetLitmus::InventoryManipulation
   include PuppetLitmus::PuppetHelpers
   include PuppetLitmus::RakeHelper
+  Honeycomb.configure do |config|
+    # config.debug = true
```

Comment:
Allow this configuration to be set from an environment variable - just in case any debugging information is needed

Reply:
@carabasdaniel sure removing it since its checking ENV value . https://github.com/honeycombio/beeline-ruby/blob/4713124437d4038c25aeaf618291aaca064cc094/lib/honeycomb/configuration.rb#L15-L21